lib/config/rules/typescript.js

```diff
@@ -10,15 +10,21 @@ module.exports = {
   // enforces interface names are prefixed. (interface-name from TSLint)
   'typescript/interface-name-prefix': 'off',
 
+  // enforces naming conventions for class members by visibility.
+  'typescript/member-naming': 'off',
+
   // enforces /// <reference /> is not used. (no-reference from TSLint)
   'typescript/no-triple-slash-reference': 'error',
 
-  // enforces the any type is not used. (no-any from TSLint)
-  'typescript/no-explicit-any': 'off',
+  // disallow generic Array constructors
+  'typescript/no-array-constructor': 'error',
```

Comment:
This is to replicate:

https://github.com/Shopify/eslint-plugin-shopify/blob/master/lib/config/rules/stylistic-issues.js#L85_L86

ESLint original rule: https://eslint.org/docs/rules/no-array-constructor
eslint-plugin-typescript: https://github.com/nzakas/eslint-plugin-typescript/blob/master/docs/rules/no-array-constructor.md

Reply:
Can we add this to the CHANGELOG, please.